fibers/scheduler.scm

```diff
@@ -173,8 +173,8 @@ with no arguments.
 This function is thread-safe even if @var{sched} is running on a
 remote kernel thread."
   (schedule-task/no-wakeup sched task)
-  (unless (eq? ((scheduler-kernel-thread sched)) (current-thread))
-    (epoll-wake! (scheduler-epfd sched)))
+  ; (unless (eq? ((scheduler-kernel-thread sched)) (current-thread))
+  ;   (libevt-wake! (scheduler-libevt sched)))
```

Comment:
i'm not sure if this is intended or not.

Reply:
I commented out those lines because I removed the wake read/write pipes as they are not necessary. We don't leave guile mode when the event loop is running, so there no need to use libevent-wake